libraries/chain/include/scorum/chain/dbs_budget.hpp

```diff
@@ -59,11 +60,7 @@ class dbs_budget : public dbs_base
 
     /** Lists all fund budgets
      */
-    budget_refs_type get_fund_budgets() const;
-
-    /** Count of all fund budgets
-     */
-    uint64_t get_fund_budget_count() const;
```

Comment:
Why u removed this method? Now we need enumerate (create vector) to get list size :(